### PR TITLE
Better documentation

### LINKED ISSUE #18: Better documentation

### PR DESCRIPTION
**Pull request by ianpogi5**
Fixes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,32 +42,138 @@ The function above will return
...
 ## API
 
-All of the methods have the same API.
+### Successful responses
+
+| Method     | Code | Description                                                                                                                 |
+| ---------- | ---- | --------------------------------------------------------------------------------------------------------------------------- |
+| OK         | 200  | Request has succeeded and the message body contains the requested information.                                              |
+| CREATED    | 201  | Request has succeeded and a new resource has been created. The message body may contain information about the new resource. |
+| NO_CONTENT | 204  | Request has succeeded but there is not content to be returned.                                                              |
```
Comment (mbungalso) [inline README.md:53]: Request has succeeded but there is **not** content to be returned.
Should be: Request has succeeded but there is **no** content to be returned.

File: README.md [hunk 2]
```diff
@@ -42,32 +42,138 @@ The function above will return
...
+
+Options
+
+| Option  | Type                   | Required                | Default | Description                                                                                                                                                                                                        |
+| ------- | ---------------------- | ----------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| body    | object, string or null | :ballot_box_with_check: | null    | <ul><li>**object** will be converted into JSON string and will have a _Content-Type_ of _application/json_ in the header.</li><li>**string** will have a a _Content-Type_ of _text/plain_ in the header.</li></ul> |
```
Comment (mbungalso) [inline README.md:59]: ... will have **a a** _Content-Type_ of ...
Should be: ... will have **a** _Content-Type_ of ...

File: README.md [hunk 3]
```diff
@@ -42,32 +42,138 @@ The function above will return
...
+  permanent: true,
+  location: 'https://www.google.com',
+});
+```
+
+### Client Error responses
+
+| Method       | Code | Description                                                                                                           |
+| ------------ | ---- | --------------------------------------------------------------------------------------------------------------------- |
+| BAD_REQUEST  | 400  | The server could not understand the request due to invalid syntax or missing paramters.                               |
```
Comment (mbungalso) [inline README.md:99]: incorrectly spelling **paramters**
Should be: **paramaters**

**Review by mbungalso — CHANGES_REQUESTED** [review]
Please fix 2 grammar error and 1 spelled word.

File: .changeset/five-planets-watch.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@kdcio/api-gw-resp': patch
+---
+
+Improve documentation
```
Comment (mbungalso) [inline .changeset/five-planets-watch.md:5]: Improve documentation
May I also know what is the intention for this text?

Comment (ianpogi5) [reply .changeset/five-planets-watch.md:5]: Para sa changeset yan

**Pull request merged**